IOPool/Input/src/PoolSource.cc

```diff
@@ -294,42 +290,6 @@ namespace edm {
   }
 
   void
-  PoolSource::readOneRandom(EventPrincipal& cache, size_t& fileNameHash, CLHEP::HepRandomEngine* engine) {
-    assert(!secondaryFileSequence_);
-    primaryFileSequence_->readOneRandom(cache, fileNameHash, engine);
-  }
-
-  bool
-  PoolSource::readOneRandomWithID(EventPrincipal& cache, size_t& fileNameHash, LuminosityBlockID const& lumiID, CLHEP::HepRandomEngine* engine) {
-    assert(!secondaryFileSequence_);
-    return primaryFileSequence_->readOneRandomWithID(cache, fileNameHash, lumiID, engine);
-  }
-
-  bool
-  PoolSource::readOneSequential(EventPrincipal& cache, size_t& fileNameHash) {
-    assert(!secondaryFileSequence_);
-    return primaryFileSequence_->readOneSequential(cache, fileNameHash);
-  }
-
-  bool
-  PoolSource::readOneSequentialWithID(EventPrincipal& cache, size_t& fileNameHash, LuminosityBlockID const& lumiID) {
-    assert(!secondaryFileSequence_);
-    return primaryFileSequence_->readOneSequentialWithID(cache, fileNameHash, lumiID);
-  }
-
-  void
-  PoolSource::readOneSpecified(EventPrincipal& cache, size_t& fileNameHash, SecondaryEventIDAndFileInfo const& id) {
-    assert(!secondaryFileSequence_);
-    primaryFileSequence_->readOneSpecified(cache, fileNameHash, id);
-  }
-
-  void
-  PoolSource::dropUnwantedBranches_(std::vector<std::string> const& wantedBranches) {
```

Comment:
No. dropUnwantedBranches_ was implemented specifically for use by the mixing module. It should not be supported for primary sources.  It remains unchanged in EmbeddedRootSource.